consul-connect/step1.md

```diff
@@ -1,31 +1,33 @@
-# View Consul UI and Launch Counting Service
+Consul is already running on a publicly accessible IP address. 
 
-We're already running Consul for you on a publicly accessible IP address. The Consul web UI runs on port `8500`. Visit it in a new tab here:
+# Access the Consul UI
+
+The Consul web UI runs on port `8500`. Visit it in a new tab here:
 
 - [Consul Web UI](https://[[HOST_SUBDOMAIN]]-8500-[[KATACODA_HOST]].environments.katacoda.com/)
 
-To start, you'll see a red `X` next to the `counting` and `dashboard` services since neither are running (so both are unhealthy).
+Since health checks have been configured, there will be a red `X` next to the `counting`, `counting-sidecar-proxy`, `dashboard`, and `dashboard-sidecar-proxy`. The services are not healthy because they are not running.
 
-<img src="https://hashicorp-education.s3-us-west-2.amazonaws.com/katacoda/consul-connect/images/1-1-web-ui.png" alt="Consul Web UI" title="Consul Web UI">
+# Start the Backend Service
 
-Now let's start some services. You'll examine the configuration for this service and then start the `counting-service` on port `9003`.
+Start the backend service, `counting-service`. First, you'll examine the configuration for this service and then start the it on port `9003`.
 
-Consul is configured to look for the `counting-service` on port `9003`. You can see the configuration by looking at the configuration file at `/etc/consul.d/counting.json`.
+The service defnition instructs Consul to look for the `counting-service` on port `9003`. You can see the service defnition by looking at the configuration file at `/etc/consul.d/counting.json`.
 
 `cat /etc/consul.d/counting.json`{{execute}}
 
-There are three important settings here.
+There are three important settings in the service definition.
 
 - Consul will look for a service running on port `9003`. It will advertise that as the `counting` service. On a properly configured node, this can be reached as `counting.service.consul` through DNS.
```

Comment:
```suggestion
- Consul will look for a service running on port `9003`. It will advertise that as the `counting` service. On a properly configured node, this can be reached as `counting.service.consul` through Consul DNS interface.
```